resources/views/ResultFields/BasketItem.fields.json

```diff
@@ -34,7 +34,9 @@
     "images.variation.path",
     "images.variation.names.alternate",
     "images.variation.names.lang",
-    "properties.*",
+    "properties.property.id",
+    "properties.property.surcharge",
+    "properties.property.isShownAtCheckout",
```

Comment:
Hier sollte aber noch properties.property.names.* mit rein

Reply:
Das Feld wird an der Stelle nicht benötigt, da im Warenkorb die Namen der Bestellmerkmale bereits unter `basketItem.basketItemOrderParams[0].name` ausgeliefert und in Ceres auch aus diesem Feld gelesen.